### PR TITLE
Playlist creation and authentication

### PR DESCRIPTION
**Pull request by mdarif6**
- Playlist Creation Done
- Like and dislike feature added
- Playlist deletion Functionality added
- Authentication added

### CODE REVIEW

File: src/context/video-context.jsx [hunk 1]
```diff
@@ -17,11 +23,87 @@ function videoReducer(state, action) {
...
+      const updatedLiked = state.liked.filter(
+        (item) => item._id == action.payload
+      );
+      return { ...state, liked: updatedLiked };
+    case "ADD_TO_PLAYLIST":
+      return {
+        ...state,
+        playlists: action.payload,
+      };
+    // case "DELETE_FROM_PLAYLIST":
```
Comment (ravikumar1002) [inline src/context/video-context.jsx]: remove this

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

**Review by PeeyushPrashant — APPROVED** [review]
code looks modular
good to merge

File: src/context/video-context.jsx (hunk 1 above)
Comment (mdarif6) [reply src/context/video-context.jsx]: done

**Pull request merged**